ServerCore/Pages/Events/CreateDemo.cshtml.cs

```diff
@@ -482,6 +487,15 @@ public async Task<IActionResult> OnPostAsync()
                 transaction.Commit();
             }
 
+            // Create base files for event content and style
+            if (AddSkeletonContentFiles)
```

Comment:
If it's efficient to put a try/catch around the NewFileHelper stuff, maybe remove this flag?

Reply:
This is the checkbox on the demo creation page that we said we wanted since demo events sometimes don't need these.

Reply:
I would argue that demo events on the server always need these, and local events on a dev box don't because some people don't configure local storage. If it gracefully failed instead, that might be enough.

Reply:
I feel like we never discussed local/dev events vs. on the server, so we're kind of getting them mixed up with demo events vs. non-demo events.  My understanding was that real/non-demo events will ALWAYS need these new files because on the server we're obviously going to use the event, and locally people would be creating a non-demo event specifically for local customization of its content.  If anything, they would create a demo event just to use the puzzles it creates, so they WOULDN'T need the homepage stuff for anything in a demo event, which is why we decided they should be optional when we talked about it.  

That's a good point though that in either case, if the event is local/dev, file creation could fail due to storage not being set up, so this will need to fail silently if that's the case.